pypeit/wavetilts.py

```diff
@@ -147,6 +170,75 @@ def spatid_to_zero(self, spat_id):
         mtch = self.spat_id == spat_id
         return np.where(mtch)[0][0]
 
+    def show(self, waveimg=None, wcs_match=True, in_ginga=True, show_traces=False):
+        """
+        Show in ginga or mpl Tiltimg with the tilts traced and fitted overlaid
+
+        Args:
+            waveimg (`numpy.ndarray`_, optional):
+                Image with the wavelength solution.
+            wcs_match (bool, optional):
+                If True, use this image as a reference image for the WCS and match all
+                image in other channels to it.
+            in_ginga (bool, optional):
+                If True, show the image in ginga. Otherwise, use matplotlib.
+            show_traces (bool, optional):
+                If True, show the traces of the tilts on the image.
+
+        """
+        # get slits
+        if (Path(self.calib_dir).resolve() / self.slits_filename).exists():
+            slits = slittrace.SlitTraceSet.from_file(Path(self.calib_dir).resolve() / self.slits_filename)
+            slitmask = slits.slit_img(initial=True, flexure=self.spat_flexure)
+            left, right, mask = slits.select_edges(flexure=self.spat_flexure)
+            gpm = mask == 0
+        else:
+            slits = None
+            msgs.warn('Could not load slits to show with tilts image.')
+
+        # get tiltimg
+        if (Path(self.calib_dir).resolve() / self.tiltimg_filename).exists():
+            tilt_img_dict = buildimage.TiltImage.from_file(Path(self.calib_dir).resolve() / self.tiltimg_filename)
+            tilt_img = tilt_img_dict.image * (slitmask > -1) if slits is not None else tilt_img_dict.image
+        else:
+            msgs.err('Tiltimg not found.')
```

Comment:
Two suggested changes `err`-> `error` and `Tiltimg` -> `Tilt image`
`msgs.error('Tilt image not found')`

Reply:
Thank you for the suggestions. Added!